man/import.q.concourse.Rd

```diff
@@ -2,19 +2,22 @@
 \alias{import.q.concourse}
 \title{Q methodology: import concourse of Q items}
 \description{
-  Imports full set of items (concourse) from a directory of *.tex files, including possible translations.
+  Imports full set of items (concourse) from a directory of .TEX files, including possible translations.
 }
 \usage{
-  import.q.concourse(q.concourse.dir, languages)
+  import.q.concourse(q.concourse.dir, languages=NULL)
 }
 
 \arguments{
   \item{q.concourse.dir}{
-    a directory of folders per language, with all full q items as *.tex files in each language folder, with files named as the same meaningful item short handle (e.g. \code{happy-feeling.tex}) across all languages.
+    A directory of individual .TEX files named as meaningful item short handles (e.g. \code{happy-feeling.tex}).
+    If \code{languages} are specified, a directory with folders per language, with all full Q items as individual .TEX files in each language folder, with files named as the \emph{same} meaningful item short handle (e.g. \code{happy-feeling.tex}) across all languages.
+
     Directories end with a trailing slash, such as \code{study/q-sample/q-concourse/}.
    }
   \item{languages}{
   	a character vector of languages, same as folders below \code{q.concourse.dir}.
+  	If your concourse is monolingual, leave empty, defaults to \code{NULL}.
   }
 }
 \details{
```

Comment:
done.
Ps.: That part is likely not shown in your view, because you're looking at the *diff*, not the entire commit.
To be able to comment on *all* the lines, you can just pick any commit (any commit also allows you to comment inline).

Anyway, problem solved.